cmake/external_gtest.cmake

```diff
@@ -24,6 +24,13 @@ include(ExternalProject)
 SET(GTEST_GIT_REPO_URL https://github.com/google/googletest.git)
 SET(GTEST_GIT_LABEL release-1.8.0)
 
+set(COMPILE_FLAGS -fPIC)
```

Comment:
This snippet shows up in multiple places. Perhaps we should make a cmake macro which sets COMPILE_FLAGS